lib/src/api/page_loader_element_interface.dart

```diff
@@ -65,17 +65,17 @@ abstract class PageLoaderElement extends PageLoaderSource {
   /// The shadow root hosted by this element.
   PageLoaderElement get shadowRoot;
 
-  /// The text content of this element.
+  /// The inner text of this element.
   ///
-  /// This should return the value of `<element>.innerText` (see
-  /// https://developer.mozilla.org/en-US/docs/Web/API/Node/innerText), but
-  /// behaviour may vary for the different pageloader implementations.
+  /// This should return the value of
+  /// [`<element>.innerText`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/innerText),
+  /// but behaviour may vary for the different pageloader implementations.
   String get innerText;
 
   /// Visible text within this element.
   ///
-  /// This should return the value of webdriver's getText() call (see
-  /// http://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/WebElement.html#getText--),
+  /// This should return the value of webdriver's
+  /// [`getText()`](http://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/WebElement.html#getText--),
```

Comment:
Same here.

Reply:
Improved, but I don't want to split the URL without a way to test it; do you know of one?

Reply:
I don't know of a way. Let's just leave it as it is then.